WcaOnRails/spec/jobs/submit_report_nag_job_spec.rb

```diff
@@ -27,4 +27,25 @@
       SubmitReportNagJob.perform_now
     end.to change { ActionMailer::Base.deliveries.length }.by(0)
   end
+
+  it "schedules report reminder email" do
+    _unscheduled_competition = FactoryBot.create :competition, starts: nil
```

Comment:
Nit: on this variable and others, could you please remove the leading `_`?
I know the original code has them, but I have no idea why this file has a totally different naming convention from the rest of the code :')

If we can avoid introduce more of these that would be awesome.

Reply:
the underscore is used to explicitly mark this variable as unused. We tell the compiler not to raise the corresponding warning this way. I can still get rid of the variable at all. But I find it useful TBH.

Reply:
Thanks I forgot about the use of `_` in that case.

In general I would get rid of an unused variable if I could (eg: by putting a comment before the bunch of unused variables if needed), but I don't feel too strongly about that.